risc-v-demo/Readme-compile.md

```diff
@@ -6,13 +6,10 @@ compile the application software and update the example webpage on the board.
 
 ## Requirements To Recompile the Example:
 
-- Use Libero/SmartHLS release 2024.1
+- Use Libero/SmartHLS release 2024.2
```

Comment:
Can we change this to be use the latest Libero/SmartHLS release?  That way we don't have to keep updating this readme.